specs/specs.bs

```diff
@@ -1559,11 +1558,11 @@ A CityJSON Extension can extend the core data model in four ways:
 <div class="note">
   While Extensions are less flexible than CityGML ADEs (inheritance and namespaces are for instance not supported, and less customisation is possible), it should be noted that the flexibility of ADEs comes at a price: the software processing an extended CityGML file will not necessarily know what structure to expect.
 
-  There is ongoing work to use the ADE schemas to automatically do this, but this currently is not supported by most software. 
+  There is ongoing work on using the ADE schemas to automatically do this, but this is currently not supported by most software. 
   Viewers might not be affected by ADEs because the geometries are usually not changed by an ADE (although they can!). 
   However, software parsing the XML to extract attributes and features might not work directly (and thus specific code would need to be written).
 
-  CityJSON Extensions are designed such that they can be read and processed by standard CityJSON software, often no changes in the parsing code is required. 
```

Comment:
This sentence was not clear either, so I tried to improve it.

Reply:
This is the only sentence in everything you fixed that I am not sure...

I think I meant that the design of Extension was made so that downstream applications can always read them. Not that they should, this could imply that someone design her Extension (say for energy) and that she should make it such that no code is required by others. The point is she cannot not do this.

Reply:
Ok noted, I will change this.